app/code/Magento/Ui/view/base/web/js/grid/filters/filters.js

```diff
@@ -404,11 +404,12 @@ define([
          */
         onBackendError: function (isError) {
             var defaultMessage = 'Something went wrong with processing the default view and we have restored the ' +
-                    'filter to its original state.',
-                customMessage  = 'Something went wrong with processing current custom view and filters have been ' +
+                'filter to its original state.',
+                customMessage = 'Something went wrong with processing current custom view and filters have been ' +
                     'reset to its original state. Please edit filters then click apply.';
 
             if (isError) {
+                console.log(this.bookmarksActiveIndex);
```

Comment:
I believe you left it for debug purposes

Reply:
yep. Forgot remove it after test